client/build.rs

```diff
@@ -0,0 +1,56 @@
+//! Build script which builds smartcontract for test
+//!
+//! Technically this script is used only for testing purposes, but current cargo implementation
+//! doesn't allow to run build script only for tests or get info about current profile from it.
+//! See [cargo issue #4001](https://github.com/rust-lang/cargo/issues/4001)
+
+use std::{env, path::Path, process::Command};
+
+#[allow(clippy::expect_used)]
+fn main() {
+    let manifest_dir =
+        env::var("CARGO_MANIFEST_DIR").expect("Expected `CARGO_MANIFEST_DIR` environment variable");
+    let smartcontract_path =
+        Path::new(&manifest_dir).join("tests/integration/create_nft_for_every_user_smartcontract");
+    let path_env = env::var("PATH").expect("Expected `PATH` environment variable");
+    let out_dir = env::var_os("OUT_DIR").expect("Expected `OUT_DIR` environment variable");
+
+    // It's better to rerun this script anytime something in the main folder is changed so that
+    // we don't have to manually monitor every iroha_wasm dependency
+    println!("cargo:rerun-if-changed=..");
+
+    let fmt = Command::new("cargo")
+        // Clearing environment variables to avoid `error: infinite recursion detected`.
+        .env_clear()
+        // Setting `PATH` variable so that [`Command`] can find `cargo`
+        .env("PATH", path_env.clone())
+        .current_dir(smartcontract_path.clone())
+        .args(&["+nightly", "fmt", "--all"])
```

Comment:
```suggestion
        .args(&["+nightly", "rustfmt", "--all"])
```

I'm not sure, but it's `rustfmt` in toolchain file

Reply:
![image](https://user-images.githubusercontent.com/15856657/164203227-5dd0816e-4c9c-46f4-8215-82f1ec365781.png)

Reply:
It should be `fmt`